integrations/coinbase/src/main/java/org/dash/wallet/integration/coinbase_integration/Constants.kt

```diff
@@ -16,23 +16,24 @@
  */
 package org.dash.wallet.integration.coinbase_integration
 
-const val CB_VERSION_KEY = "CB-VERSION"
-const val CB_2FA_TOKEN_KEY = "CB-2FA-TOKEN"
-const val CB_VERSION_VALUE = "2021-09-07"
-const val TRANSACTION_TYPE_SEND = "send"
-const val TRANSACTION_STATUS_PENDING = "pending"
-const val TRANSACTION_STATUS_COMPLETED = "completed"
-const val TRANSACTION_STATUS_FAILED = "failed"
-const val TRANSACTION_STATUS_EXPIRED = "expired"
-const val TRANSACTION_STATUS_CANCELED = "canceled"
-const val TRANSACTION_STATUS_WAITING_FOR_SIGNATURE = "waiting_for_signature"
-const val TRANSACTION_STATUS_WAITING_FOR_CLEARING = "waiting_for_clearing"
-const val DASH_CURRENCY = "DASH"
-const val USD_CURRENCY = "USD"
-const val ERROR_ID_INVALID_REQUEST = "invalid_request"
-const val ERROR_ID_2FA_REQUIRED = "two_factor_required"
-const val ERROR_MSG_INVALID_REQUEST = "That code was invalid"
-const val MIN_SWAP_DASH_AMOUNT_VALUE = "0.01909299"
-const val VALUE_ZERO = "0"
-const val DEFAULT_CURRENCY_USD = "USD"
-const val MIN_USD_COINBASE_AMOUNT = "2"
\ No newline at end of file
+object CoinbaseConstants {
```

Comment:
It's better to group constants to an object. Keeps it more structured in case we want to reference these in the wallet module, plus no need to import each one separately.